e2e/src/helper.ts

```diff
@@ -315,3 +372,33 @@ export function ensureDirectoryExists(dir: string): void {
     fs.mkdirSync(dir);
   }
 }
+
+export function waitForLanguageServerReady(projectFolderName: string): Promise<void> {
+  let lsReadyDeferred = languageServerReady.get(projectFolderName);
+  if (lsReadyDeferred === undefined) {
+    lsReadyDeferred = deferred<void>();
+    languageServerReady.set(projectFolderName, lsReadyDeferred);
+  }
+  return lsReadyDeferred.promise;
+}
+
+export function initializeExtensionApi(): void {
+  const dbtLanguageServer = extensions.getExtension('fivetran.dbt-language-server');
+  if (dbtLanguageServer) {
+    extensionApi = dbtLanguageServer.exports as ExtensionApi;
```

Comment:
Probably it is better to use dbtLanguageServer.activate() method because extension should be activated first. It will return API.

Also we can rename method to initializeExtension and get rid of duplicated code from `activateAndWait` function:
`extensions.getExtension('fivetran.dbt-language-server');`
I'm not sure that we need to call `await ext.activate();` every time from `activateAndWait`.